tests/schemas/api_v3/transaction.schema.js

```diff
@@ -46,8 +57,8 @@ const transactionSchema = {
 	sender: Joi.object(sender).required(),
 	params: Joi.object().required(),
 	block: Joi.object(block).required(),
-	confirmations: Joi.number().required(),
 	executionStatus: Joi.string().valid(...TRANSACTION_EXECUTION_STATUSES).required(),
+	meta: Joi.object(transactionMetaSchema).required(),
```

Comment:
I think `transactionMetaReceipientSchema`is optional. Please verify and update. 
```suggestion
	meta: Joi.object(transactionMetaSchema).optional(),
```

Reply:
Applied.